packages/fluentui/react-northstar/src/components/Box/Box.tsx

```diff
@@ -28,7 +28,7 @@ export interface BoxProps extends UIComponentProps<BoxProps>, ContentComponentPr
   /** Accessibility behavior if overridden by the user. */
   accessibility?: Accessibility<never>;
 }
-export type BoxStylesProps = never;
+export type BoxStylesProps = {};
```

Comment:
Merging with `never` will not work as any union that contains `never` becomes `never` 😨